lib/kubernetes-deploy/errors.rb

```diff
@@ -0,0 +1,3 @@
+module KubernetesDeploy
+  class FatalDeploymentError < StandardError; end
```

Comment:
Is it already defined in https://github.com/Shopify/kubernetes-deploy/blob/master/lib/kubernetes-deploy.rb#L12 ?